### DISCUSSION TITLE
Ridurre tempo di aggiornamento posizione

### DISCUSSION THREAD

**Discussion by dragonblu19** [indomus/forum#1738] [HUB: Home Assistant]
Salve a tutti, chiedo aiuto. In pratica ho creato delle azioni che si attivano in base alla posizione. Qui sorge il problema.
In pratica per quello che ho potuto trovare credo di aver concesso tutti i permessi, ma non sono riuscito a ridurre il tempo di aggiornamento della posizione. La posizione in home assistant mi si aggiorna ogni 5 minuti. Quindi potete capire che se ad esempio la posizione si aggiorna poco prima di entrare nell'area, devo aspoettare 5 minuti che si aggiorni la posizione.
C'è quindi un modo per ridurre questo tempo?

Grazie.

**Comment by 50494554524F** [comment]
prova con

https://www.home-assistant.io/integrations/homeassistant/#service-homeassistantupdate_entity

**Reply by KC-inDomus** [reply]
non c'entra nulla. l'aggiornamento è solo a carico dell'app Companion e del funzionamento intrinseco del sistema operativo dello smartphone su cui gira.
